front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3340,6 +3340,10 @@ export type SimulationSummaryResult =
   | (PathfindingInputError & {
       status: 'pathfinding_input_error';
     });
+export type PacedTrainSimulationSummaryResult = {
+  exceptions: SimulationSummaryResult[];
```

Comment:
If I'm correct, this part should be more like that
```suggestion
  exceptions: {[key: string]: SimulationSummaryResult};
```
Where the keys would be each `key` of the exceptions ?

Reply:
Sorry, yes